packages/vitepress-plugin-og-image-generator/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@nolebase/vitepress-plugin-og-image-generator",
+  "type": "module",
+  "version": "1.0.0",
+  "description": "A vitepress plugin to generate Open Graph Protocol previewing images (a.k.a. social media cards) for your site.",
+  "author": {
+    "name": "Nólëbase",
+    "email": "nolebase@ayaka.moe",
+    "url": "https://github.com/nolebase"
+  },
+  "license": "MIT",
+  "sideEffects": false,
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.js"
+    }
+  },
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "README.md",
+    "dist",
+    "package.json"
+  ],
+  "scripts": {
+    "dev": "concurrently \"pnpm run stub\"",
+    "stub": "unbuild --stub",
+    "build": "unbuild"
+  },
+  "peerDependencies": {
+    "vitepress": ">=1.0.0-rc.44"
+  },
+  "devDependencies": {
+    "@resvg/resvg-js": "^2.6.0",
+    "@types/fs-extra": "^11.0.4",
+    "emoji-regex": "^10.3.0",
+    "fast-glob": "^3.3.2",
+    "fs-extra": "^11.2.0"
+  }
+}
```

Comment:
Please add an empty line at the end of the file.

If you use VSCode, you can configure these two for better development experience:

```json
    "files.insertFinalNewline": true,
    "files.trimTrailingWhitespace": true,
```